package.json

```diff
@@ -67,7 +67,17 @@
         }
       ]
     },
+    "fileAssociations": {
+        "ext": "slp",
+        "name": "slp",
+        "description": "Slippi File Format",
+        "icon": "./resources/icon.ico"
```

Comment:
could probably add `"role": "viewer"` here too (not sure what it does but we did it for URI's, it's mac only)